docs/guide/custom-ui.md

```diff
@@ -198,6 +198,46 @@ The `AposWidget` component has **nothing to do with a typical site visitor exper
 Before you override an editor modal, consider [adding a custom schema field type](/guide/custom-schema-field-types.md) instead.
 :::
 
+## Adding custom context menu
+
+Starting from Apostrophe v3.18.0 we can add custom context menus (edit mode) from within any module, targeting any Vue component that implements `AposModal`. The menu registration should happen in the initialization phase.
```

Comment:
It is unclear what this actually does and where this menu appears. Please include a screenshot.